applications/asset_tracker_v2/src/modules/data_module.c

```diff
@@ -94,7 +94,8 @@ static struct cloud_data_cfg current_cfg = {
 	.accelerometer_inactivity_threshold	= CONFIG_DATA_ACCELEROMETER_INACT_THRESHOLD,
 	.accelerometer_inactivity_timeout	= CONFIG_DATA_ACCELEROMETER_INACT_TIMEOUT_SECONDS,
 	.no_data.gnss		 = !IS_ENABLED(CONFIG_DATA_SAMPLE_GNSS_DEFAULT),
-	.no_data.neighbor_cell	 = !IS_ENABLED(CONFIG_DATA_SAMPLE_NEIGHBOR_CELLS_DEFAULT)
+	.no_data.neighbor_cell	 = !IS_ENABLED(CONFIG_DATA_SAMPLE_NEIGHBOR_CELLS_DEFAULT),
+	.no_data.wifi		 = !IS_ENABLED(CONFIG_DATA_SAMPLE_WIFI_DEFAULT)
```

Comment:
This `.no_data.wifi` line is something we should add to fix a bug with WiFi build.
If discussions lead to removal of `CONFIG_DATA_SAMPLE_WIFI_DEFAULT` from this PR, I guess this could be:
`.no_data.wifi = !IS_ENABLED(CONFIG_LOCATION_METHOD_WIFI)`

Reply:
By default without setting specific configurations, we should leave `.no_data.wifi` false just like other `no_data` variables.
The main reason for this is that if we set this to true, it will be written to cloud and when you build and flash WiFi config, WiFi positioning will be unintentionally disabled in the cloud.